docs/kyma/docs/017-details-local-installation.md

```diff
@@ -0,0 +1,93 @@
+---
+title: Local installation
+type: Details
+---
+
+Simple as it may seem, the Kyma installation is a complex and dynamic process. This document extends the “031-gs-local-installation.md” guide with a detailed breakdown of the alternative installation method, i.e. the “run.sh” script.
+
+> **NOTE:** the ‘run.sh’ script should be used exclusively for development purposes.
+
+To start the local installation, run the following command:
+```
+./installation/cmd/run.sh
+```
+
+This script sets up default parameters, starts Minikube, builds Kyma-Installer, generates local configuration, creates the `installation` Custom Resource and sets up Installer. Subsequent sections provide a detailed description of each step.
+
+You can execute the `installation/cmd/run.sh` script with the following parameters:
+
+- `--skip-minikube-start` which skips execution of the `installation/scripts/minikube.sh` script. See the **Start Minikube** section for more details
+- `--vm-driver` points to either `virtualbox` or `hiperkit` depending on your operating system
+
+The following snippet lies in the heart of the “run.sh” script:
+
+```
+if [[ ! ${SKIP_MINIKUBE_START} ]]; then
+    bash ${CURRENT_DIR}/../scripts/minikube.sh --domain "${DOMAIN}" --vm-driver "${VM_DRIVER}"
+fi
+
+bash ${CURRENT_DIR}/../scripts/build-kyma-installer.sh --vm-driver "${VM_DRIVER}"
+
+bash ${CURRENT_DIR}/../scripts/generate-local-config.sh
+
+if [ -z "$CR_PATH" ]; then
+
+    TMPDIR=`mktemp -d "${CURRENT_DIR}/../../temp-XXXXXXXXXX"`
+    CR_PATH="${TMPDIR}/installer-cr-local.yaml"
+
+    bash ${CURRENT_DIR}/../scripts/create-cr.sh --output "${CR_PATH}" --domain "${DOMAIN}"
+    bash ${CURRENT_DIR}/../scripts/installer.sh --local --cr "${CR_PATH}"
+
+    rm -rf $TMPDIR
+else
+    bash ${CURRENT_DIR}/../scripts/installer.sh --cr "${CR_PATH}"
+fi
+```
+
+The subsequent paragraphs describe respective sub-scripts triggered during the installation process.
+
+## The minikube.sh script
+
+> **NOTE:** To work with Kyma, use only the provided scripts and commands. Kyma does not work on a basic Minikube cluster that you can start using the `minikube start` command or stop with the `minikube stop` command. If you don't need Kyma on Minikube anymore, remove the cluster with the `minikube delete` command.
+
+Purpose of `installation/scripts/minikube.sh` is to configure and start Minikube. The script also checks if your development environment is cofigured to handle Kyma installation. This includes checking Minikube and Kubectl versions. If Minikube is already initialized, the system prompts you to agree to remove the previous Minikube cluster. The script exits if you don't want to restart your cluster.
```

Comment:
- Purpose -> The purpose
- Kyma installation -> the Kyma installation
- Kubectl -> kubectl

Reply:
done